src/Core/ELILO.pm

```diff
@@ -172,6 +172,7 @@ sub GetMetaData() {
 	noedd30		=> "bool:Don't force EDD30 Mode if not set:",
 	fpswa		=> "path:Specify the Filename for a specific FPSWA to load:",
 	"secure-boot"	=> "string:Support Secure Boot:auto",
+	multipath	=> "int:Number of additional multipath entries:0",
```

Comment:
that is strange, as pbl should use cached __lines and remove or replace only changed parts, but maybe there is some magic in yast2-bootloader as SLE11 still use quite strange concept of metadata.